alertwildfire_live_recording.py

```diff
@@ -143,31 +138,90 @@ def fetchAllCameras(googleServices, camera_names_to_watch):
 
 
 
+def cleanup_archive(googleServices, dbManager, timethreshold):
+    """initializes a continual cleaning function that only acts to remove archived data past a given threshold.
+    Args:
+        googleServices: Drive service (from getGoogleServices())
+        dbManager: Database service
+        timethreshold (flt): hours to keep data in archive
+    Returns:
+        continual should never return must be manually killed
+    """
+    while True:
```

Comment:
Why make this a forever loop vs. one-time run?  If you are going to keep it as a forever loop, you should at least add a sleep in there (maybe 10 minute?)